zookeeper-server/src/main/java/org/apache/zookeeper/server/ServerMetrics.java

```diff
@@ -526,5 +526,21 @@ public void resetAll() {
     public MetricsProvider getMetricsProvider() {
         return metricsProvider;
     }
+    // make it worse
+    public void dummy() {
+        if (true) {
+            if (true) {
+                if (true) {
+                    if (true) {
+                    }
+                }
+            }
+        }
+        if (true) {
+            if (true) {
+
+            }
+        }
```

Comment:
**Warning**
New issue: Bumpy Road Ahead

dummy has 2 logical blocks with deeply nested code. Threshold is one single block per functio

Reply:
**❌ New issue: Bumpy Road Ahead**
dummy has 2 logical blocks with deeply nested code. Threshold is one single block per function

Reply:
**❌ New issue: Bumpy Road Ahead**
dummy has 2 logical blocks with deeply nested code. Threshold is one single block per function